### ISSUE TITLE
Feature request - device type 

### ISSUE THREAD

**Issue by laurenssss** [robertklep/name.klep.sonoff#19]
When I add for instance a green wave socket to homey I get a selection if the device is a normal outlet, light or ventilator etc

Can this be added to the pairing wizard for sonoff as well?

**Comment by robertklep** [comment]
Good idea, although I'll wait for Homey v2.0 before I implement this, since 2.0 might have other (new) device classes.

**Comment by laurenssss** [comment]
Can imagine, thanks for the app by the way, wizard works perfect with just delivered basic devices
